packages/salesforcedx-utils-vscode/test/jest/config/configUtil.test.ts

```diff
@@ -0,0 +1,39 @@
+import { SfProject } from '@salesforce/core';
+import { ConfigUtil, workspaceUtils } from '../../../src';
+
+describe('ConfigUtil', () => {
+  describe('getProjectPackageNames', () => {
+    const mockSfProject: any = {
+      getUniquePackageNames: jest
+        .fn()
+        .mockImplementation(() => ['project1', 'project2'])
+    };
```

Comment:
This mock of the getUniquePackageNames fn doesn't seem to be returning the mock impl values when it gets called when debugging the test